ui/app/modifiers/code-mirror.js

```diff
@@ -29,7 +29,10 @@ export default class CodeMirrorModifier extends Modifier {
 
   @action
   _onChange(editor) {
-    this.args.named.onUpdate(editor.getValue(), this._editor);
+    // avoid sending change event after initial setup when editor value is set to content
+    if (this.args.named.content !== editor.getValue()) {
+      this.args.named.onUpdate(editor.getValue(), this._editor);
+    }
```

Comment:
@Monkeychip this was this issue causing the same computation error. Since we are setting the editor value to the content arg in the setup the change event was firing immediately.